lib/super_digit.rb

```diff
@@ -1,15 +1,21 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
You would need to specify what `n` is.  It could be the number of digits or the size of n.